qiskit/algorithms/time_evolvers/variational/__init__.py

```diff
@@ -0,0 +1,119 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2023.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+Variational Quantum Time Evolutions (:mod:`qiskit.algorithms.time_evolvers.variational`)
+========================================================================================
+
+Algorithms for performing Variational Quantum Time Evolution of quantum states,
+which can be tailored to near-term devices.
+:class:`~qiskit.algorithms.time_evolvers.variational.VarQTE` base class exposes an interface, compliant
+with the Quantum Time Evolution Framework in Qiskit Terra, that is implemented by
+:class:`~qiskit.algorithms.VarQRTE` and :class:`~qiskit.algorithms.VarQITE` classes for real and
+imaginary time evolution respectively. The variational approach is taken according to a variational
+principle chosen by a user.
+
+Examples:
+
+.. code-block::python
```

Comment:
Not 100% sure but I think this misses an indentation, as this is not rendered in the docs: 
<img width="847" alt="image" src="https://user-images.githubusercontent.com/5978796/213004909-7f1320de-7a39-4c6e-a18a-2c4493e6360d.png">